### ISSUE TITLE
Error on install

### ISSUE THREAD

**Issue by m49n** [RomainMazB/octobercms-frontend-model-editor#2]
Update failed
"SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'json not null, `displayed_actions` json not null) default character set utf8mb4 ' at line 1 (SQL: create table `romainmazb_femodeleditor_models` (`id` bigint unsigned not null auto_increment primary key, `namespace` varchar(191) not null, `model_name` varchar(191) not null, `url_param` varchar(191) null default 'id', `link_text` varchar(191) not null default 'model', `pages_names` json not null, `displayed_actions` json not null) default character set utf8mb4 collate utf8mb4_unicode_ci engine = InnoDB)" on line 664 of /vendor/laravel/framework/src/Illuminate/Database/Connection.php

**Comment by RomainMazB** [comment]
Hi, again thanks to report the issue.

Since I didn't get this error, I tried to test the SQL Syntax provided in the error message using [EverSQL Syntax Validator](https://www.eversql.com/sql-syntax-check-validator/) which doesn't return any error:
![Capture d’écran de 2020-05-24 23-22-33](https://user-images.githubusercontent.com/53976837/82765277-a51f1180-9e15-11ea-80ba-5a1918cba731.png)

I'm using it in local and two web environment and it works perfectly. I've just tried on a fresh install, and it also worked.

Knowing that, all I can tell is: Did you try to uninstall/reinstall it?
